tests/views/test_message.py

```diff
@@ -18,6 +19,13 @@ def setUp(self):
         self.app = app.test_client()
         self.app.testing = True
         app.config.from_object(TestingConfig)
+        self.before()
+
+    @requests_mock.mock()
```

Comment:
Are we going to use requests_mock or responses going forward? My vote goes for responses, it seems to be much more fully featured and comes from Sentry, so should be decent quality: https://github.com/getsentry/responses

Reply:
We'll discuss in the tech session on Monday then. For now I've followed the convention in the rest of the code.